docker/worker/run_openqa_worker.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-/usr/share/openqa/script/worker --verbose --instance "$OPENQA_WORKER_INSTANCE"
+/root/qemu/kvm-mknod.sh
+su _openqa-worker -c /usr/share/openqa/script/worker --verbose --instance "$OPENQA_WORKER_INSTANCE"
```

Comment:
this again isn't really covered by the git commit message.

Reply:
> kvm operations cannot be instanced during the building because the workers may have kvm but the build machine don't have to

This seems to cover it

Reply:
oh, I see. Because we want to call /root/qemu/kvm-mknod.sh as root we need to remove the user setting in Dockerfile and instead only call the worker script with the user. Yeah, that works :)